### ISSUE TITLE
Add issue type for Housekeeping

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#1349]
**Please add a short description of your enhancement request**

I'd like another issue type for Housekeeping.

**Describe the solution you'd like**

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or enhancement you've considered.

**Additional context**

### Kick Off:

- [ ] Ensure the enhancement is `UX refined` and aligned with UX 
_The component and/or enhancement should be published and available in the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)_
- [ ] Ensure the enhancement has been `Tech refined` with @kirbydesign/kirby-guild and this issue is updated with a clear implementation description  
_This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation)_
- [ ] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code:

- [ ] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
- [ ] Create a draft implementation and push to Github
- [ ] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request
- [ ] Implement unit tests
- [ ] Update Cookbook Examples and Showcase, i.e. see [Radio](https://cookbook.kirby.design/home/showcase/radio)  
_Also remember to add any relevant new API documentation_

### Review:

- UX review:
  - [ ] Ensure implementation is correct in relation to the UX design and the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)
  - [ ] With UX agree on the version of the implementation
- Code review:
  - [ ] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review  
_Remember to add `closes #issueno` to the description of the PR._
  - [ ] Once approved, merge feature branch/PR to master
- [ ] Ask a member of @kirbydesign/kirby-guild to add a link to component showcase from Kirby Component Status and update the version number

:tada: Celebrate
